composer.json

```diff
@@ -7,6 +7,34 @@
         "coding standard"
     ],
     "require": {
-        "squizlabs/php_codesniffer": "^2.7"
+        "squizlabs/php_codesniffer": "3.0.1"
```

Comment:
So what if there are bug fix releases? We have to wait until someone updates this repository while it's already fixed in phpcs?

If you want to restrict this, I would use `~3.0.1`. It will get the bug fix releases and sticks to the 3.0.x branch.